Source/panels/charpanel.cpp

```diff
@@ -269,42 +265,45 @@ void DrawStatButtons(const Surface &out)
 
 void LoadCharPanel()
 {
-	LoadArt("data\\charbg.pcx", &PanelFull);
-	UpdatePalette(&PanelFull); // PanelFull is being used as a render target
-	PanelBoxLeft = LoadClx("data\\boxleftend.clx");
-	PanelBoxMiddle = LoadClx("data\\boxmiddle.clx");
-	PanelBoxRight = LoadClx("data\\boxrightend.clx");
-
-	const Surface out(PanelFull.surface.get());
-
-	const bool isSmallFontTall = IsSmallFontTall();
-	const int attributeHeadersY = isSmallFontTall ? 112 : 114;
-	for (unsigned i : AttributeHeaderEntryIndices) {
-		panelEntries[i].position.y = attributeHeadersY;
-	}
-	panelEntries[GoldHeaderEntryIndex].position.y = isSmallFontTall ? 105 : 106;
+	OptionalOwnedClxSpriteList background = LoadOptionalClx("data\\charbg.clx");
+	if (!background)
+		app_fatal(_("Please update devilutionx.mpq to the latest version"));

```

Comment:
Is this specifically because `charbg.clx` was missed in the first round? I don't think we should really give error messages for in between version changes, we don't support development versions in general. So maybe check should be in the init where devilutionx.mpq is loaded, similar to how diabdat.mpq is tested for by probing title.pcx.

Reply:
Done